js/toolbar/z-plane-list.js

```diff
@@ -97,6 +97,8 @@ function ZPlaneList(parent) {
      */
     this.validFetchRequests = ['fetchZPlaneListByZIndex', 'fetchZPlaneList'];
 
+    this.defaultZIndex = -1;
```

Comment:
comment what this is (like the previous ones)

```
/**
* DESCRIPTION GOES HERE
* @type {integer}
*/
```

Reply:
Done